spec/models/testimonial_spec.rb

```diff
@@ -12,5 +12,41 @@
 require 'rails_helper'
 
 RSpec.describe Testimonial, type: :model do
-  pending "add some examples to (or delete) #{__FILE__}"
+  subject{build(:testimonial)}
```

Comment:
Muy bueno esto, ahora se podría simplificar un poco con `shoulda matchers`:
[https://github.com/thoughtbot/shoulda-matchers#on-the-subject-of-subject](https://github.com/thoughtbot/shoulda-matchers#on-the-subject-of-subject)

El listado de matchers disponibles:
[https://github.com/thoughtbot/shoulda-matchers#activemodel-matchers]
(https://github.com/thoughtbot/shoulda-matchers#activemodel-matchers)

Listado de matchers disponibles:
[https://github.com/thoughtbot/shoulda-matchers#activemodel-matchers](https://github.com/thoughtbot/shoulda-matchers#activemodel-matchers)

Reply:
Si! Tal cual. Despues para el ticket de Announcements lo implemente.